resources/styles/components/learning-guide.less

```diff
@@ -30,6 +30,7 @@
     margin: 0 auto;
 
     svg {
+      font-size:12px;
```

Comment:
That way Fred can globally change the rem size and make everything smaller/bigger